### PR TITLE
Translation from CarmendeSantiago/TEI

### PR DESCRIPTION
**Pull request by CarmendeSantiago**

### CODE REVIEW

File: P5/Source/Specs/abbr.xml [hunk 1]
```diff
@@ -127,6 +134,8 @@
         </valItem>
         <valItem ident="title">
           <gloss xml:lang="en" versionDate="2017-06-25">title</gloss>
+          <gloss xml:lang="es" versionDate="2022-03-16">la abreviatura se emplea para un
```
Comment (HelenaSabel) [inline P5/Source/Specs/abbr.xml]:
```suggestion
          <gloss xml:lang="es" versionDate="2022-03-16">la abreviatura se emplea para una
```

File: P5/Source/Specs/abbr.xml [hunk 2]
```diff
@@ -127,6 +134,8 @@
...
+tratamiento en la dirección (el Dr.,, Sr.,…)</gloss>
```
Comment (HelenaSabel) [inline P5/Source/Specs/abbr.xml]:
```suggestion
fórmula de tratamiento (Dr, Dra, Sra, Sr, …)</gloss>
```

File: P5/Source/Specs/abbr.xml [hunk 3]
```diff
@@ -53,6 +52,8 @@
       <valList type="open" mode="add">
         <valItem ident="suspension">
           <gloss xml:lang="en" versionDate="2017-06-25">suspension</gloss>
+          <gloss xml:lang="es" versionDate="2022-03-16">la abreviatura proporciona a las
```
Comment (HelenaSabel) [inline P5/Source/Specs/abbr.xml]:
```suggestion
          <gloss xml:lang="es" versionDate="2022-03-16">la abreviatura proporciona las
```

**Review by HelenaSabel — CHANGES_REQUESTED** [review]
There were small issues in the translation and I commited the required changes. The PR is ready to merge.

**Comment by rettinghaus** [comment]
I think all these `gloss` should be `desc`.

**Comment by sydb** [comment]
Even though my Spanish is _way_ too weak to contribute meaningfully, I think @rettinghaus is correct — these should probably be descriptions, not glosses. Except that `@type` of `<abbr>` _already has_ a gloss in Spanish (“tipo”), and it _already has_ a description in Spanish, that looks suspiciously like the glosses provided on this PR. Has someone already fixed this and not closed the PR?

**Comment by HelenaSabel** [comment]
There was confusion between `<desc>` and `<gloss>` and the translation could be improved a bit (like trying to not use the so called “generic masculine”. Thus, we decided to just close this PR.

**Pull request closed without merging**